alma/lib/Utils/PaymentDataHelper.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * 2018-2022 Alma SAS
+ *
+ * THE MIT LICENSE
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+ * Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+ * WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ *
+ * @author    Alma SAS <contact@getalma.eu>
+ * @copyright 2018-2022 Alma SAS
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ */
+
+namespace Alma\PrestaShop\Utils;
+
+use Alma\PrestaShop\Model\AddressData;
+use Alma\PrestaShop\Model\CartData;
+use Alma\PrestaShop\Model\CustomerData;
+use Alma\PrestaShop\Model\ShippingData;
+use Cart;
+use Context;
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+/**
+ * Class PaymentDataHelper.
+ *
+ * Payment Data
+ */
+class PaymentDataHelper
+{
+    public function __construct(
+        Cart $cart,
+        Context $context,
+        array $feePlan
+    ) {
+        $customerData = new CustomerData($context, $cart);
+
+        $this->cart = $cart;
+        $this->context = $context;
+        $this->feePlan = $feePlan;
+        $this->customer = $customerData->get();
+    }
+
+    /**
+     * Return data for payment
+     *
+     * @return array paymentData
+     */
+    public function paymentData()
+    {
+        $purchase_amout = CartData::getPurchaseAmount($this->cart);
+        $addressData = new AddressData($this->cart);
+        $customerHelper = new CustomerHelper($this->customer, $this->context, $this->cart);
+        $shippingAddress = $addressData->getShippingAddress();
+        $billingAddress = $addressData->getBillingAddress();
+
+        $dataPayment = [
```

Comment:
Call the variable you want to return the same as you method

```suggestion
        $paymentData = [
```